app/models/build.py

```diff
@@ -38,6 +38,8 @@ def __init__(
         :param defconfig_full: The full value of the defconfig when it contains
         fragments. Default to the same 'defconfig' value.
         :type defconfig_full: string
+        :param build_environment: The docker image used to build the kernel_image
```

Comment:
Rather than explicitly mentioning Docker, it may be better to say something more generic such as "description of the build environment used to build the kernel_image" or something like that.  We may not always use Docker, and it would be good if this field could still be used with other build environments.

Reply:
Done